SpywareFilter/sections/tracking_servers_firstparty.txt

```diff
@@ -26,6 +26,8 @@
 ! Disabled in DNS filter
 ||telemetry.mozilla.org^
 !
+||s.news.naver.com^
+||cc.naver.com^
```

Comment:
> ||cc.naver.com^

I don't see it. But If I remember, it can break redirects on this site.

Reply:
Then, what do you think of this?
```suggestion
||cc.naver.com/cc
```

Reply:
Please show full URL and where can I see it.

Reply:
URL: `https://section.blog.naver.com/BlogHome.nhn?directoryNo=0&currentPage=1&groupId=0`
If you click the highlighted rectangle box, you can see the logs like under picture.
![network](https://user-images.githubusercontent.com/37463155/101654560-95c18780-3a38-11eb-9a2c-5d9076fc3ae1.png)
![Screenshot](https://user-images.githubusercontent.com/37463155/101654547-91956a00-3a38-11eb-9541-9b6a772dd561.png)

Reply:
Ok, `||cc.naver.com/cc` or `||cc.naver.com/cc?a=`

Reply:
> Then, what do you think of this?
> ```suggestion
> ||cc.naver.com/cc
> ```

I want to do this.
Do you agree?

Reply:
Updated at 3ce7a9cc71811a294faf52d3a8ef24c116352a9f.